<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraView</class>
 <widget class="QWidget" name="CameraView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>相机</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0" rowspan="2">
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter_U">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGraphicsView" name="graphicsView_L"/>
      <widget class="QGraphicsView" name="graphicsView_R"/>
     </widget>
     <widget class="QSplitter" name="splitter_D">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGraphicsView" name="graphicsView_RGB"/>
      <widget class="QGraphicsView" name="graphicsView_Depth"/>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_CameraOn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>开</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_CameraOff">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>关</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_Shot">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>拍照</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Device Avaliable</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>350</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame_4">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_FindSavePath">
        <property name="text">
         <string>保存到</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_SavePath">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>计数器:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_FindSavePath</sender>
   <signal>clicked()</signal>
   <receiver>CameraView</receiver>
   <slot>OnFindSavePath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_CameraOn</sender>
   <signal>clicked()</signal>
   <receiver>CameraView</receiver>
   <slot>OnCameraPowerOn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>569</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_CameraOff</sender>
   <signal>clicked()</signal>
   <receiver>CameraView</receiver>
   <slot>OnCameraPowerOff()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>665</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>691</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Shot</sender>
   <signal>clicked()</signal>
   <receiver>CameraView</receiver>
   <slot>OnShot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>712</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnFindSavePath()</slot>
  <slot>OnCameraPowerOn()</slot>
  <slot>OnCameraPowerOff()</slot>
  <slot>OnFocus()</slot>
  <slot>OnShot()</slot>
  <slot>OnValueChanged_ExposureCompensation(int)</slot>
  <slot>OnValueChanged_Quality(int)</slot>
  <slot>OnSelectedChanged_CameraDevice(QString)</slot>
  <slot>OnSelectedChanged_Resolution(QString)</slot>
  <slot>OnSelectedChanged_Codec(QString)</slot>
 </slots>
</ui>
